tests/e2e_tests/l2b_to_distortion_e2e.py

```diff
@@ -0,0 +1,113 @@
+import argparse
+import os
+import numpy as np
+from corgidrp.data import AstrometricCalibration
+
+import corgidrp.mocks as mocks
+import corgidrp.astrom as astrom
+import corgidrp.walker as walker
+import pytest
+import glob
+
+
+thisfile_dir = os.path.dirname(__file__) # this file's folder
+
+@pytest.mark.e2e
+def test_l2b_to_distortion(e2eoutput_path):
+    '''
+
+    An end-to-end test that creates L2b level data and runs it through the astrometric calibration distortion map solution step.
+
+        It checks that: 
+            - Distortion map coefficients are recorded correctly
+            - The distortion map error in the central 1" x 1" detector region is < 4 [mas] (~0.1835 [pixel])
+
+        Data needed: 
+            - L2b level dataset - created in the test
+    
+    Args:
+        e2eoutput_path (str): Path to the output directory
+
+
+    '''
+
+    e2eoutput_path = os.path.join(e2eoutput_path, "l2b_to_distortion_output")
+    if not os.path.exists(e2eoutput_path):
+        os.mkdir(e2eoutput_path)
+
+    e2e_data_path = os.path.join(e2eoutput_path, "astrom_distortion")
+    if not os.path.exists(e2e_data_path):
+        os.mkdir(e2e_data_path)
+
+
+    #################################
+    #### Generate a mock dataset ####
+    #################################
+
+    field_path = os.path.join(os.path.dirname(__file__),"..","test_data", "JWST_CALFIELD2020.csv")
+    distortion_coeffs_path = os.path.join(os.path.dirname(__file__),"..","test_data", "distortion_expected_coeffs.csv")
+
+    #Create the mock dataset
+    mock_dataset = mocks.create_astrom_data(field_path=field_path, filedir=e2e_data_path, rotation=20, distortion_coeffs_path=distortion_coeffs_path, dither_pointings=3)
```

Comment:
this function creates a dataset with L3 headers. Can we make sure the input dataset has L2b headers (so no WCS entry)?